modules/infrastructure-components.adoc

```diff
@@ -1,21 +1,19 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
-// * post_installation_configuration/cluster-tasks.adoc
 // * nodes-nodes-creating-infrastructure-nodes.adoc
 
 [id="infrastructure-components_{context}"]
 = {product-title} infrastructure components
 
-The following infrastructure workloads do not incur {product-title} worker subscriptions:
+To qualify as an infrastructure node and use the included entitlement, only components that are supporting the cluster, and not part of an end-user application, can run on those instances. Examples include:
```

Comment:
Wording taken from second paragraph of [Self-managed Red Hat OpenShift sizing and subscription guide -> Infrastructure nodes](https://www.redhat.com/en/resources/self-managed-openshift-sizing-subscription-guide?extIdCarryOver=true&sc_cid=701f2000001OH6kAAG#section-4).